### DISCUSSION TITLE
How can I validate template syntax in Go code?

### DISCUSSION THREAD

**Discussion by Caratpine** [projectdiscovery/nuclei#5272] [Q&A]
Hi team, I try to read some related code,  but I don't know which code I should use? 
My requirement is to check the syntax of a template. Is there such a function? Thanks

**Comment by rxerium** [comment]
Hey @Caratpine you may want to try the -validate flag like so:
```
nuclei -t template.yaml -validate
```

This will tell you where the issue lies in your template.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
